Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.8 as base
+
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+FROM base as builder
+
+WORKDIR /install
+
+COPY . .
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements-server.txt
```

Comment:
I think you can drop these lines. The pip selector below, `[…]`, is equivalent. It reads these files and installs the dependencies they list.